scripts/common_startup.sh

```diff
@@ -9,6 +9,7 @@ done
 # Conda Python is in use, do not use virtualenv
 if python -V 2>&1 | grep -q -e 'Anaconda' -e 'Continuum Analytics' ; then
     SET_VENV=0
+    CONDA_ALREADY_INSTALLED=1
```

Comment:
Well, the Conda check is done here, but the echo statement is not: `You have Conda`, but `Conda affects VirtualVenv`, which is done later on in the code. So we are printing something that is not a concern...yet, right?